.github/workflows/ci.yaml

```diff
@@ -10,36 +10,16 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/composite/go-setup
-      - name: Lint
-        run: make check && make check-templates
-      - name: Check that 'func.yaml schema' is up-to-date
-        run: make schema-check
-      - name: Check embedded templates content
-        run: go test -run "^\QTestFileSystems\E$/^\Qembedded\E$" ./pkg/filesystem
+      - name: Lint Core
+        run: make check
+      - name: Lint Templates
+        run: make check-templates
+      - name: Check 'func.yaml' schema
+        run: make check-schema
+      - name: Check embedded FS
+        run: make check-embedded-fs
 
-  test-unit:
```

Comment:
I would not recommend deleting these.

Reply:
Will `make test-all` run really all tests? What about coverage have you ensured that there are no conflicts?